deployability/modules/testing/tests/helpers/utils.py

```diff
@@ -1,234 +1,9 @@
-        status = agentd_output_decoded.split('=')[1].replace("'", "").strip()
-
-    return status
-
-
-def get_file_encoding(file_path: str) -> str:
-    """Detect and return the file encoding.
-
-    Args:
-        file_path (str): File path to check.
-
-    Returns:
-        encoding (str): File encoding.
-    """
-    with open(file_path, 'rb') as f:
-        data = f.read()
-        if len(data) == 0:
-            return 'utf-8'
-        result = chardet.detect(data)
-    return result['encoding']
-
-
-def file_monitor(monitored_file: str, target_string: str, timeout: int = 30) -> None:
-    """
-    Monitor a file for a specific string.
-
-    Args:
-        monitored_file (str): The file to monitor.
-        target_string (str): The string to look for in the file.
-        timeout (int, optional): The time to wait for the string to appear in the file. Defaults to 30.
-
-    Returns:
-        None: Returns None if the string is not found within the timeout.
-        str: Returns the line containing the target string if found within the timeout.
-    """
-    encoding = get_file_encoding(monitored_file)
-
-    # Check in the current file content for the string.
-    with open(monitored_file, encoding=encoding) as _file:
-        for line in _file:
-            if target_string in line:
-                return line
-
-    # Start count to set the timeout.
-    start_time = time.time()
-
-    # Start the file monitoring for future lines.
-    with open(monitored_file, encoding=encoding) as _file:
-        # Go to the end of the file.
-        _file.seek(0, 2)
-        while time.time() - start_time < timeout:
-            current_position = _file.tell()
-            line = _file.readline()
-
-            if not line:
-                # No new line, wait for nex try.
-                _file.seek(current_position)
-                time.sleep(0.1)
-            else:
-                # New line, check if the string matches.
-                if target_string in line:
-                    return line
-
-
-def check_agent_is_connected(agent_id: str, timeout: int = 60) -> bool:
-    """
-    Wait for an agent to connect to the manager, returns true when it does.
-
-    Args:
-        agent_id (str): The ID of the agent to wait for.
-
-    Returns:
-        bool: True if the agent connects within the timeout, False otherwise.
-    """
-    start_time = time.time()
-
-    while time.time() - start_time < timeout:
-        status = get_agent_connection_status(agent_id)
-        if status in ["connected", "Active"]:
-            return True
-        time.sleep(1)
-
-    raise False
-
-
-def read_json_file(filepath):
-    with open(filepath) as f_json:
-        return json.load(f_json)
+import yaml
+
+class Utils:
+    
+    @staticmethod
+    def extract_ansible_host(file_path):
+        with open(file_path, 'r') as yaml_file:
+            inventory_data = yaml.safe_load(yaml_file)
+        return inventory_data.get('ansible_host')
```

Comment:
```suggestion
        return inventory_data.get('ansible_host')

```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/993a6b591ec5c2172b9a6195de7fb9cae9401d14a